orangewidget/report/report.py

```diff
@@ -432,6 +433,8 @@ def plural(s, number, suffix="s"):
     :type suffix: str
     :rtype: str
     """
+    warnings.warn("Plural formed by this function is difficult to translate. "
+                  "Use orangecanvas.utils.localization.pl instead.")
```

Comment:
Do we plan to leave this function here and always warn people about using it or do we plan to eventually remove it?
In the latter case I would make it more explicit that it is deprecated and will be removed in the future (when).